src/components/Avatar/__test__/Avatar.test.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import Avatar from "../"
+import { render, cleanup, fireEvent } from "react-testing-library"
+import { ThemeProvider } from "../../../common/styles"
+import "jest-styled-components"
+import "jest-dom/extend-expect"
+
+afterEach(cleanup)
+
+const defaultProps = {
+  size: "24px",
+  disabled: false,
+  onClick: () => {},
+}
+
+const additionalProps = {
```

Comment:
In additional to snapshots which are good safety nets to prevent unwanted changes we should also put in some explicit tests to ensure that the behaviour is as per expectation.